src/sage/functions/orthogonal_polys.py

```diff
@@ -974,7 +974,7 @@ def __init__(self):
             sage: chebyshev_U(x, x)._sympy_()
             chebyshevu(x, x)
             sage: maxima(chebyshev_U(2,x, hold=True))
-            3*((-(8*(1-_SAGE_VAR_x))/3)+(4*(1-_SAGE_VAR_x)^2)/3+1)
+            3*(...-...(8*(1-_SAGE_VAR_x))/3)+(4*(1-_SAGE_VAR_x)^2)/3+1)
```

Comment:
```
sage: maxima(chebyshev_U(2,x, hold=True)).sage()
4*(x - 1)^2 + 8*x - 5
```
I don't know what this is trying to acomplish... I guess that maxima knows how to handle  `chebyshev_U(2, x)`.

Reply:
I guess it's testing conversion to maxima, so I'm not sure converting back to sage the result makes much sense

Reply:
Yeah, it's supposed to be an example of the `__init__` method. Why send it to maxima at all?